templates/index.html

```diff
@@ -2,4 +2,40 @@
 
 {% block content %}
   <h1>Welcome to Canonical</h1>
+
+  <footer class="p-footer is-dark">
+    <div class="row">
+      <hr />
+    </div>
+    <div class="row">
+      <div class="col-8">
+        <p>&copy; {{ current_year }} Canonical Ltd.<br />
+          <a href="/contact-us" class="p-footer__link">Contact information</a> |
+          <a href="https://ubuntu.com/legal" class="p-footer__link">Legal information</a> |
+          <a href="https://github.com/canonical-websites/canonical.com/issues/new" class="p-footer__link">Improve this site</a>
+        </p>
+      </div>
+      <div class="col-4 u-align--right">
+        <ul class="p-inline-list">
+          <li class="p-inline-list__item">
+              <a href="http://www.facebook.com/ubuntucloud" class="p-icon--facebook"></a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="http://www.twitter.com/ubuntucloud" class="p-icon--twitter"></a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="http://youtube.com/jujucharms" class="p-icon--youtube"></a>
```

Comment:
This doesn't feel like a good fit either - can't find a Conical Youtube page so I'd recommend removing this icon also.

Reply:
Haha, good spot. All copied from jaas.ai.